Tests/Marketplace/upload_packs.py

```diff
@@ -552,12 +562,13 @@ def main():
             pack.cleanup()
             continue
 
-        task_status = pack.remove_unwanted_files()
+        task_status = pack.remove_unwanted_files(remove_test_playbooks)
```

Comment:
this is also don't related to your PR

Reply:
actually i added this, it is a flag to set if not to remove test playbooks from the bucket